core/src/main/java/nl/nn/adapterframework/errormessageformatters/FixedErrorMessageFormatter.java

```diff
@@ -50,17 +50,16 @@ public Message format(String errorMessage, Throwable t, INamedObject location, M
 		if (messageToReturn.isEmpty()) {
 			messageToReturn=new Message("");
 		}
-		if (StringUtils.isNotEmpty(getFileName())) {
+		if (StringUtils.isNotEmpty(getFilename())) {
 			try {
-				messageToReturn = new Message(messageToReturn.asString() + Misc.resourceToString(ClassUtils.getResourceURL(this, getFileName()), Misc.LINE_SEPARATOR));
+				messageToReturn = new Message(messageToReturn.asString() + Misc.resourceToString(ClassUtils.getResourceURL(this, getFilename()), Misc.LINE_SEPARATOR));
 			} catch (Throwable e) {
-				log.error("got exception loading error message file [" + getFileName() + "]", e);
+				log.error("got exception loading error message file [{}]", getFilename(), e);
```

Comment:
Are you sure this works? I think it might try to tostring the exception and 'try' (but fail) to append the result to the log-string

Reply:
Yes, the very last element can be a Throwable. If the number of arguments is more than the placeholders in the message ParameterizedMessage will use the last as throwable.

Reply:
Oh cool!